examples/seismic/tutorials/13_LSRTM_acoustic.ipynb

```diff
@@ -468,41 +468,40 @@
     "image = np.zeros((model0.vp.shape[0], model0.vp.shape[1]))\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    nrec=101
</pre><p>Needs a space for pep8</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>